divisible.rb

```diff
@@ -0,0 +1,5 @@
+def divisible
+array = (1..100).to_a
+array.select { |item| item % 2 == 0 || item % 3 == 0 || item % 5 == 0 }
```

Comment:
Indentation inside a function or a block is important. It should look better this way and it would be easier to read.
![indentationdiv](https://user-images.githubusercontent.com/47392623/93381225-35de8a80-f815-11ea-9968-3a2282190ab3.png)